docs/How_to_Guides/How_to_Reprocess_a_Previously_Recorded_FreeMoCap_Recording_Session.md

```diff
@@ -0,0 +1,11 @@
+## HOW TO REPROCESS A PREVIOUSLY RECORDED `FreeMoCap` RECORDING SESSION
+
+You can re-start the processing pipeline from any of the following processing stages (defined below)by specifying the `SessionID` desired `stage` in the call to `freemocap.RunMe()`
+
+So to process the session named `sesh_2021-11-21_19_42_07` starting from stage 3 (aka, skipping the `1- recording` and `2- synchronization` stages), run:
+```python
+import freemocap
+freemocap.RunMe(sessionID="sesh_2021-11-21_19_42_07", stage=3)
+```
```

Comment:
I would also add a note saying that you can use the `setDataPath = True` flag to bring up a GUI to choose the path to your data folder, in case you have different folders in different areas